src/jquery.ime.js

```diff
@@ -551,9 +552,9 @@
 		// But for complex scripts, browsers place cursor in unexpected places
 		// and it's not possible to fix cursor programmatically.
 		// Ref Bug https://bugs.webkit.org/show_bug.cgi?id=66630
-		element.value = element.value.substring( 0, start - precedingCharCount ) +
+		element.value = element.value.slice( 0, start - precedingCharCount ) +
 			newText +
-			element.value.substring( element.selectionEnd, element.value.length );
+			element.value.slice( element.selectionEnd, element.value.length );
```

Comment:
Automatic change by eslint, which requires slice() instead of substring() and substr().